bundles/CoreBundle/src/EventListener/Frontend/ContentTemplateListener.php

```diff
@@ -91,14 +90,18 @@ public function onKernelView(ViewEvent $event): void
         );
     }
 
-    private function resolveParameters(ControllerArgumentsEvent $event, ?array $vars): array
+    private function resolveParameters(ViewEvent $event, ?array $vars): array
```

Comment:
```suggestion
    private function resolveParameters(ViewEvent $event, array $vars): array
```
`array_merge` and `array_keys` doesn't accept null